packages/opentelemetry-host-metrics/src/enum.ts

```diff
@@ -27,6 +27,11 @@ export enum METRIC_NAMES {
   PROCESS_MEMORY_USAGE = 'process.memory.usage',
 }
 
+export enum METRIC_ATTRIBUTES {
```

Comment:
I've added this enum to properly report the attributes with their full name as set on the semantic convettions

See: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemcputime